src/test/java/edu/brown/cs/student/main/StarsJUnitTest.java

```diff
@@ -0,0 +1,58 @@
+package edu.brown.cs.student.main;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StarsJUnitTest {
+
+  @Test
+  public void testOverlappingStars() {
+    StarDistance distCalculator = new StarDistance();
+    Double[] pos1 = new Double[3];
+    pos1[0] = 0.0;
+    pos1[1] = 0.0;
+    pos1[2] = 0.0;
+    Double[] pos2 = new Double[3];
+    pos2[0] = 0.0;
+    pos2[1] = 0.0;
+    pos2[2] = 0.0;
+    double output = distCalculator.euclideanDistance(pos1, pos2);
+    assertEquals(0.0, output, 0.01);
+  }
+
+  @Test
```

Comment:
i would include a comment explaining this test as it is not immediately apparent